src/components/pages/deployments/deploymentsHome/deployments.js

```diff
@@ -57,15 +57,23 @@ export class Deployments extends Component {
 
   onGridReady = gridReadyEvent => this.deploymentGridApi = gridReadyEvent.api;
 
+  getSoftSelectId = ({ id } = '') => id;
```

Comment:
I think this is what you meant `({ id = '' })`

Reply:
done